docs/architecture/diagrams/src/FAC_Sequence.puml

```diff
@@ -0,0 +1,61 @@
+@startuml
+Actor User as Grantee
+Actor User as Auditor
+Actor User as Public
+Actor User as Agency
+
+Participant “FAC.gov” as FAC
+Participant “Login.gov” as LoginGov
+Participant “Data.gov” as DataGov
+
+Database “DataStore” as DB
+Database "S3" as S3
+
+autonumber 
+== Submission ==
+Grantee -> FAC : Grantee Accesses FAC
+FAC -> LoginGov : Redirects for authentication
+LoginGov -> FAC : Redirects to FAC
+Grantee -> FAC : Fetches Submission
+DB -> FAC : Fetches previously saved structured data
+S3 -> FAC : Fetches previously saved xlsx docs
```

Comment:
“docs” here should be “documents”, or “documents” elsewhere should be “docs”, for consistency.